ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -305,6 +305,11 @@ def pytest_addoption(parser):
         loaded when run-ci starts
         """,
     )
+    parser.addoption(
+        "--regional-dr",
```

Comment:
Please change to something related to import as this is suppose to be only for import OCP clusters to ACM.

Reply:
done